ci/infra/testrunner/utils/utils.py

```diff
@@ -52,9 +52,11 @@ def cleanup_file(file):
         if os.path.exists(file):
             logger.info(f"Cleaning up {file}")
             try:
-                if os.path.isfile(file):
+                try:
+                    # Attempt to remove the file first, because a socket (e.g.
+                    # ssh-agent) is not a file but has to be removed like one.
                     os.remove(file)
-                else:
+                except IsADirectoryError:
```

Comment:
Where is the defintion for this?

I am getting:

```
Starting ./testrunner script
Traceback (most recent call last):
  File "/home/tux/go/src/github.com/SUSE/skuba/ci/infra/testrunner/testrunner.py", line 12, in <module>
    import platforms
  File "/home/tux/go/src/github.com/SUSE/skuba/ci/infra/testrunner/platforms/__init__.py", line 1, in <module>
    from platforms.openstack import Openstack
  File "/home/tux/go/src/github.com/SUSE/skuba/ci/infra/testrunner/platforms/openstack.py", line 4, in <module>
    from platforms.terraform import Terraform
  File "/home/tux/go/src/github.com/SUSE/skuba/ci/infra/testrunner/platforms/terraform.py", line 6, in <module>
    from platforms.platform import Platform
  File "/home/tux/go/src/github.com/SUSE/skuba/ci/infra/testrunner/platforms/platform.py", line 6, in <module>
    from utils import (step, Utils)
  File "/home/tux/go/src/github.com/SUSE/skuba/ci/infra/testrunner/utils/__init__.py", line 4, in <module>
    from utils.utils import (Utils, step)
  File "/home/tux/go/src/github.com/SUSE/skuba/ci/infra/testrunner/utils/utils.py", line 58
    except IsADirectoryError:
         ^
SyntaxError: invalid syntax
```

Reply:
`IsADirectoryError` is a built-in error in Python3 - https://docs.python.org/3/library/exceptions.html#IsADirectoryError

Which version of `python3` are you using to run the script?

Reply:
My bad, it works now.